<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 120, 120); color: white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,13">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>工作日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnUploadFile">
     <property name="text">
      <string>上传文件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnUploadFolder">
     <property name="text">
      <string>上传文件夹</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDownloadFile">
     <property name="text">
      <string>下载文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
